routes/userRoutes.js

```diff
@@ -52,70 +55,67 @@ router.patch('/:id', requireLogin, async (req, res) => {
     }
 
     if (validFields) {
-      const updatedUser = await User.findByIdAndUpdate( id, { $set: updateFields, modifiedAt: Date.now() }, { new: true })
+      // Extracts the points and level fields from updateFields
```

Comment:
This is something we need to look at before we can merge this branch. The current implementation on dev is not passing tests correctly with this implementation. We have created a fix for it, that needs implementing. (let us talk about it tomorrow. 18.10/2023